package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "focus-lock",
-  "version": "0.8.1",
+  "name": "@kulevw/focus-lock",
+  "version": "0.8.2",
```

Comment:
please do not change version manually. (as well as change the name of this package)